### ISSUE TITLE
Dell megaraid (older machine) not seen

### ISSUE THREAD

**Issue by kgk** [coreos/bugs#2131]
# Issue Report #
### Container Linux Version ###

```
 $  cat /etc/os-release
NAME="Container Linux by CoreOS"
ID=coreos
VERSION=1465.6.0
VERSION_ID=1465.6.0
BUILD_ID=2017-08-16-0012
PRETTY_NAME="Container Linux by CoreOS 1465.6.0 (Ladybug)"
ANSI_COLOR="38;5;75"
HOME_URL="https://coreos.com/"
BUG_REPORT_URL="https://issues.coreos.com"
COREOS_BOARD="amd64-usr"

```

What hardware/cloud provider/hypervisor is being used to run Container Linux?

Dell 1850/1950

```
$ lsblk
NAME  MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
sr0    11:0    1  1024M  0 rom  
loop0   7:0    0 251.2M  0 loop /usr
 $ lspci -nn
00:00.0 Host bridge [0600]: Intel Corporation E7520 Memory Controller Hub [8086:3590] (rev 09)
00:02.0 PCI bridge [0604]: Intel Corporation E7525/E7520/E7320 PCI Express Port A [8086:3595] (rev 09)
00:04.0 PCI bridge [0604]: Intel Corporation E7525/E7520 PCI Express Port B [8086:3597] (rev 09)
00:05.0 PCI bridge [0604]: Intel Corporation E7520 PCI Express Port B1 [8086:3598] (rev 09)
00:06.0 PCI bridge [0604]: Intel Corporation E7520 PCI Express Port C [8086:3599] (rev 09)
00:1d.0 USB controller [0c03]: Intel Corporation 82801EB/ER (ICH5/ICH5R) USB UHCI Controller #1 [8086:24d2] (rev 02)
00:1d.1 USB controller [0c03]: Intel Corporation 82801EB/ER (ICH5/ICH5R) USB UHCI Controller #2 [8086:24d4] (rev 02)
00:1d.2 USB controller [0c03]: Intel Corporation 82801EB/ER (ICH5/ICH5R) USB UHCI Controller #3 [8086:24d7] (rev 02)
00:1d.7 USB controller [0c03]: Intel Corporation 82801EB/ER (ICH5/ICH5R) USB2 EHCI Controller [8086:24dd] (rev 02)
00:1e.0 PCI bridge [0604]: Intel Corporation 82801 PCI Bridge [8086:244e] (rev c2)
00:1f.0 ISA bridge [0601]: Intel Corporation 82801EB/ER (ICH5/ICH5R) LPC Interface Bridge [8086:24d0] (rev 02)
00:1f.1 IDE interface [0101]: Intel Corporation 82801EB/ER (ICH5/ICH5R) IDE Controller [8086:24db] (rev 02)
01:00.0 PCI bridge [0604]: Intel Corporation 80332 [Dobson] I/O processor (A-Segment Bridge) [8086:0330] (rev 06)
01:00.2 PCI bridge [0604]: Intel Corporation 80332 [Dobson] I/O processor (B-Segment Bridge) [8086:0332] (rev 06)
02:0e.0 RAID bus controller [0104]: Dell PowerEdge Expandable RAID controller 4 [1028:0013] (rev 06)
04:00.0 PCI bridge [0604]: Intel Corporation 80332 [Dobson] I/O processor (A-Segment Bridge) [8086:0330] (rev 07)
04:00.2 PCI bridge [0604]: Intel Corporation 80332 [Dobson] I/O processor (B-Segment Bridge) [8086:0332] (rev 07)
05:0e.0 RAID bus controller [0104]: LSI Logic / Symbios Logic MegaRAID [1000:0408] (rev 07)
07:00.0 PCI bridge [0604]: Intel Corporation 6700PXH PCI Express-to-PCI Bridge A [8086:0329] (rev 09)
07:00.2 PCI bridge [0604]: Intel Corporation 6700PXH PCI Express-to-PCI Bridge B [8086:032a] (rev 09)
08:07.0 Ethernet controller [0200]: Intel Corporation 82541GI Gigabit Ethernet Controller [8086:1076] (rev 05)
09:08.0 Ethernet controller [0200]: Intel Corporation 82541GI Gigabit Ethernet Controller [8086:1076] (rev 05)
0b:0d.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] RV100 [Radeon 7000 / Radeon VE] [1002:5159]
```

**Comment by bgilbert** [comment]
Looks like we would need to enable `MEGARAID_MM` and `MEGARAID_MAILBOX`, neither of which require firmware.

**Comment by bgilbert** [comment]
This should be fixed in 1520.1.0, due shortly, and 1535.0.0.  Thanks for reporting.
